antwar.config.js

```diff
@@ -20,114 +19,116 @@ module.exports = {
-    },
-    api: {
-      title: 'API',
-      url: ({ sectionName, fileName }) => `/${sectionName}/${fileName}/`,
-      layout: () => require('./src/components/Page/Page.jsx').default,
-      content: () => require.context('./loaders/page-loader!./src/content/api', false, /^\.\/.*\.md$/),
-      redirects: {
-        'passing-a-config': 'configuration-types'
-      }
-    },
-    guides: {
-      title: 'Guides',
-      url: ({ sectionName, fileName }) => `/${sectionName}/${fileName}/`,
-      layout: () => require('./src/components/Page/Page.jsx').default,
-      content: () => require.context('./loaders/page-loader!./src/content/guides', false, /^\.\/.*\.md$/),
-      redirects: {
-        'code-splitting-import': '/guides/code-splitting',
-        'code-splitting-require': '/guides/code-splitting',
-        'code-splitting-async': '/guides/code-splitting',
-        'code-splitting-css': '/guides/code-splitting',
-        'code-splitting-libraries': '/guides/code-splitting',
-        'why-webpack': '/comparison',
-        'production-build': '/guides/production'
-      }
-    },
-    plugins: {
-      title: 'Plugins',
-      url: ({ sectionName, fileName }) => `/${sectionName}/${fileName}/`,
-      layout: () => require('./src/components/Page/Page.jsx').default,
-      content: () => {
-        return combineContexts(
-          require.context('./loaders/page-loader!./src/content/plugins', false, /^\.\/.*\.md$/),
-          require.context('./loaders/page-loader!./generated/plugins', false, /^\.\/.*\.md$/)
-        );
-      }
-    },
-    loaders: {
-      title: 'Loaders',
-      url: ({ sectionName, fileName }) => `/${sectionName}/${fileName}/`,
-      layout: () => require('./src/components/Page/Page.jsx').default,
-      content: () => {
-        return combineContexts(
-          require.context('./loaders/page-loader!./src/content/loaders', false, /^\.\/.*\.md$/),
-          require.context('./loaders/page-loader!./generated/loaders', false, /^\.\/.*\.md$/)
-        );
-      }
-    },
-    contribute: {
-      title: 'Contribute',
-      url: ({ sectionName, fileName }) => `/${sectionName}/${fileName}/`,
-      layout: () => require('./src/components/Page/Page.jsx').default,
-      content: () => require.context('./loaders/page-loader!./src/content/contribute', false, /^\.\/.*\.md$/),
-    },
-    vote: () => require('./src/components/Vote/Vote.jsx').default,
-    organization: () => require('./src/components/Organization/Organization.jsx').default,
-    'starter-kits': () => require('./src/components/StarterKits/StarterKits.jsx').default,
+      },
+      paths: {
 
-    /*************************
-     Redirects for Old Content
-     *************************/
-    'get-started': {
-      hidden: true,
-      redirects: {
-        '': '/guides/getting-started',
-        'install-webpack': '/guides/installation',
-        'why-webpack': '/guides/why-webpack',
-      }
+				/*************************
```

Comment:
@bebraw can you please review if redirects belong here? putting outside of parent breaks on `getAllPages` call

Reply:
It depends on the version you are using. I dropped direct support for redirects recently. I use [redirect-webpack-plugin](https://www.npmjs.com/package/redirect-webpack-plugin) instead so maybe you want to migrate to that. It's possible to do that even with an older version of Antwar although the new one is nicer (faster etc.).

Reply:
Note that the newest version of Antwar runs on webpack 4.

Reply:
Thats exactly what im doing here, moving to antwar 0.28.3 and to webpack 4, so i will use redirect-webpack-plugin instead, thank you!